### DISCUSSION TITLE
the-first-blog

### DISCUSSION THREAD

**Discussion by Tung-hehe** [Tung-hehe/personnel-website#5] [Announcements]
Comment for [/blog/the-first-blog](https://www.tungtt.dev/blog/the-first-blog)

**Comment by Tung-hehe** [comment]
Comment đầu tiên trên blog đầu tiên 😊

**Reply by Tung-hehe** [reply]
Reply đầu tiên cho comment đầu tiên trên bài viết đầu tiên 😊

**Comment by mquangdo** [comment]
hi

**Comment by Tung-hehe** [comment]
Gần 2 năm rồi nè

**Reply by mquangdo** [reply]
omg

**Reply by Tung-hehe** [reply]
Độ này sẽ viết nhiều hơn, Quang rảnh thì ghé qua nhé. Đang làm lại hệ thống comment khác nữa, ko dùng giscus nữa :v

**Reply by mquangdo** [reply]
thi thoảng em vẫn vào blog anh đọc, cảm giác như vừa học optima lab hôm qua vậy
